### PR TITLE
Tests for Badge.tsx and Loader.tsx

### PR DESCRIPTION
**Pull request by yawMD**
### Pull Request Checklist: 

- [ ] Please make sure all jobs pass before requesting a review. 
- [ ] Assign yourself to the PR
- [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
- [ ] Label it as "needs-review"
- [ ] Have you lint your code locally before submission?
- [ ] Did you write tests where appropiate?

### CODE REVIEW

File: CommonUI/src/Tests/Loader.test.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+describe('Loader tests', () => {
+    test('it should render if bar loader show up', () => {
+        render(
+            <Loader
+                size={50}
+                color={new Color('#000000')}
+                loaderType={LoaderType.Bar}
+            />
+        );
+        expect(BarLoader).toBeInTheDocument;
```
Comment (simlarsen) [inline CommonUI/src/Tests/Loader.test.tsx:18]: isnt this a function?

File: CommonUI/src/Tests/Loader.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    });
+    test('it should render if beats loader show up', () => {
+        render(
+            <Loader
+                size={50}
+                color={new Color('#000000')}
+                loaderType={LoaderType.Beats}
+            />
+        );
+        expect(BeatLoader).toBeInTheDocument;
```
Comment (simlarsen) [inline CommonUI/src/Tests/Loader.test.tsx:28]: isnt this a function?

**Review by simlarsen — CHANGES_REQUESTED** [review]

File: CommonUI/src/Tests/Loader.test.tsx (hunk 1 above)
Comment (yawMD) [reply CommonUI/src/Tests/Loader.test.tsx:18]: yeah they are

File: CommonUI/src/Tests/Loader.test.tsx (hunk 2 above)
Comment (yawMD) [reply CommonUI/src/Tests/Loader.test.tsx:28]: sure is

File: CommonUI/src/Tests/Loader.test.tsx (hunk 1 above)
Comment (simlarsen) [reply CommonUI/src/Tests/Loader.test.tsx:18]: then why dont you call it? toBeInTheDocument()

Comment (yawMD) [reply CommonUI/src/Tests/Loader.test.tsx]: changed

**Review by simlarsen — APPROVED** [review]

**Pull request merged**